public/data/bisesh.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Bisesh Adhikari",
+  "type": "personal",
+  "bio": "Student | Tech enthusiast | Aspiring software developer",
+  "avatar":  "https://github.com/Biseshadhikari.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Biseshadhikari",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/BiseshAdhikar10",
+      "icon": "Twitter"
+    },
```

Please remove the trailing comma on line 16
```suggestion
    }
```

---

Thanks for the correction, I made the change.